biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.css

```diff
@@ -12,12 +12,17 @@ br {
 
 .buildings {
   display: inline-block;
-  margin-right: 1.25rem;
   margin-bottom: 1.25rem;
+  margin-left: 1.25rem;
+  margin-top: 1.25rem;
+  margin-right: 1.25rem;
```

Comment:
instead of defining margin bottom, left, top, and right all separately, you can use `margin: 1.25rem;`

Reply:
Ahh that makes sense